tests/queries/0_stateless/02751_multiquery_with_argument.sh

```diff
@@ -13,6 +13,10 @@ $CLICKHOUSE_LOCAL --multiquery "SELECT 200; S" 2>&1 | grep -o 'Syntax error'
 $CLICKHOUSE_LOCAL --multiquery "; SELECT 201;" 2>&1 | grep -o 'Empty query'
 $CLICKHOUSE_LOCAL --multiquery "; S; SELECT 202" 2>&1 | grep -o 'Empty query'
 
+# Simultaneous --queries-file + --query (multiquery) is prohibited.
```

Comment:
Cosmetic: let's add `passing` between `Simultaneously` and `--queries-file`.

Reply:
Done.